src/Gyro.cpp

```diff
@@ -34,67 +42,88 @@ void Gyro::startup()
     Wire.write(MPU6050_REG_WHO_AM_I);
     Wire.endTransmission(true);
     Wire.requestFrom(MPU6050_I2C_ADDR, 1, 1);
-    byte id = (Wire.read() >> 1) & 0x3F;    
+    byte id = Wire.read();
```

Comment:
```C++
const byte id = ((Wire.read() >> 1) & 0x3F);
isPresent = (id == 0x34);
```
?
There are wayyyy better ways to do register reading/comparisons but this can still be cleaned up a tad

Reply:
This used to be on one line, but I split it for debugging purposes.... is there a request here?

Reply:
In my opinion it's a much cleaner implementation, shows the intent a lot better, const-correctness, etc. Up to you in the end though :upside_down_face:

Reply:
Oh, I didn't realize that was new code you wrote, lol. Like I said while trying to get this running, I wanted to see what actually was read before the bit fiddling.. But now that it works, I'll change it to the more optimized now.